arex-storage-web-api/src/main/java/com/arextest/storage/beans/StorageAutoConfiguration.java

```diff
@@ -46,7 +56,22 @@ public StorageAutoConfiguration(StorageConfigurationProperties configurationProp
     @Bean
     @ConditionalOnMissingBean(MongoDatabase.class)
     public MongoDatabase mongoDatabase() {
-        return MongoDbUtils.create(properties.getMongodbUri());
+        MongoDatabase database = MongoDbUtils.create(properties.getMongodbUri());
+        setTtlIndexes(database);
+        return database;
+    }
+
+    private void setTtlIndexes(MongoDatabase mongoDatabase) {
+        for (MockCategoryType category : MockCategoryType.DEFAULTS) {
+            setTTLIndexInMockerCollection(category, mongoDatabase);
+        }
+    }
+
+    private void setTTLIndexInMockerCollection(MockCategoryType category, MongoDatabase mongoDatabase) {
+        String categoryName = ProviderNames.DEFAULT + category.getName() + COLLECTION_PREFIX;
+        MongoCollection<AREXMocker> collection = mongoDatabase.getCollection(categoryName, AREXMocker.class);
+        collection.createIndex(new Document(EXPIRATION_TIME_COLUMN_NAME, 1),
+            new IndexOptions().expireAfter(0L, TimeUnit.SECONDS));
```

Comment:
尝试下如果索引已经存在，然后修改成不同的ttl，看下是否能正常工作

Reply:
You cannot use [createIndex()](https://www.mongodb.com/docs/v5.0/reference/method/db.collection.createIndex/#mongodb-method-db.collection.createIndex) to change the value of expireAfterSeconds of an existing index. Instead use the [collMod](https://www.mongodb.com/docs/v5.0/reference/command/collMod/#mongodb-dbcommand-dbcmd.collMod) database command in conjunction with the [index](https://www.mongodb.com/docs/v5.0/reference/command/collMod/#mongodb-collflag-index) collection flag. Otherwise, to change the value of the option of an existing index, you must drop the index first and recreate.

Reply:
added try-catch to drop and reCreate index